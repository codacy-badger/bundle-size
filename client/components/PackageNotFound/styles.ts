import css from 'styled-jsx/css';

export const styles = css`
  .PackageNotFound__Title {
    font-size: 2rem;
    color: red;
  }
`;
