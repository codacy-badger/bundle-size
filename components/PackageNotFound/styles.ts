import css from 'styled-jsx/css';

export const styles = css`
  .PackageNotFound__Title {
    font-size: 4rem;
  }
`;
