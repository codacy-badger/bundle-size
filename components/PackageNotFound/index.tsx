import { styles } from './styles';

export default function () {
  return (
    <>
      <div className="PackageNotFound">
        <h2 className="PackageNotFound__Title">Package not found</h2>
      </div>
      <style jsx>{styles}</style>
    </>
  );
}
