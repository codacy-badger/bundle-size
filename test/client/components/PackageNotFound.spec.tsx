import React from 'react';
import { render } from '../../testUtils';
import PackageNotFound from '../../../client/components/PackageNotFound';

describe('PackageNotFound component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<PackageNotFound />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
