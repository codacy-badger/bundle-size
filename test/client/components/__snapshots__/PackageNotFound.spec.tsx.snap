// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PackageNotFound component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="jsx-3984282531 PackageNotFound"
  >
    <h2
      class="jsx-3984282531 PackageNotFound__Title"
    >
      Package not found
    </h2>
  </div>
</DocumentFragment>
`;
